javascript/index.js

```diff
@@ -15,3 +15,128 @@ mobileMenuItems.forEach((item) => {
     document.querySelector('.mobile-menu').classList.remove('show-menu');
   });
 });
+
+// Popup window
+
+const portfolio = document.querySelector('#portfolio');
+
+const projectArray = [
+  {
+    mobileImg: 'images/tonic-section.svg',
+    desktopImg: 'images/desktop-snap3.png',
+    cardTitle: 'Tonic',
+    cardDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required',
+    class: 'card',
+  },
+
+  {
+    mobileImg: 'images/availability-section.svg',
+    desktopImg: 'images/desktop-snap2.png',
+    cardTitle: 'Multi-Post Stories',
+    cardDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required',
+    class: 'card-2',
+  },
+
+  {
+    mobileImg: 'images/tonic-section.svg',
+    desktopImg: 'images/desktop-snap4.png',
+    cardTitle: 'Tonic',
+    cardDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required',
+    class: 'card',
+  },
+
+  {
+    mobileImg: 'images/tonic-section.svg',
+    desktopImg: 'images/desktop-snap3.png',
+    cardTitle: 'Tonic',
+    cardDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required',
+    class: 'card-2',
+  },
+];
+
+portfolio.innerHTML = projectArray.map((project) => {
```

Comment:
- [ ] please fix the `left,right,up,down` scrolling of your mobile popup, make it full height. also, ensure it looks almost the same as the figma version. 
your version 👇🏾 
![Screenshot (159)_LI](https://user-images.githubusercontent.com/65298048/174262828-e74d3a19-94cd-451e-a770-fcbede06798d.jpg)
figma version 👇🏾 

![Screenshot (158)](https://user-images.githubusercontent.com/65298048/174262999-97814a9e-efcd-407d-8778-39a91a5c6d6d.png)